File: Source/Rio.Models/DataTransferObjects/Parcel/ParcelMonthlyEvapotranspirationDto.cs
namespace Rio.Models.DataTransferObjects.Parcel
{
    public class ParcelMonthlyEvapotranspirationDto
    {
        public int ParcelID { get; set; }
        public string ParcelNumber { get; set; }
        public int WaterYear { get; set; }
        public int WaterMonth { get; set; }
        public decimal EvapotranspirationRate { get; set; }
    }
}

File: Source/Rio.Web/src/app/shared/models/parcel/parcel-monthly-evapotranspiration-dto.ts
export class ParcelMonthlyEvapotranspirationDto {
    ParcelID: number;
    ParcelNumber: string;
    WaterYear: number;
    WaterMonth: number;
    EvapotranspirationRate: number;
    OverriddenEvapotranspirationRate?: number;
    IsEmpty: boolean;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}